api/context.go

```diff
@@ -3,9 +3,8 @@ package api
 import (
 	"context"
 	"net/http"
-	"strings"
 
-	"github.com/qri-io/qri/repo"
+	"github.com/qri-io/qri/lib"
```

Comment:
well shoot, we should just move this entire file into lib in a future PR

Reply:
Lol, yes, just added a todo for that. Should happen rather soon.